### PR TITLE
Bio1 2000 q1+2

### PR DESCRIPTION
**Pull request by RedMincrafter**
I did bio1 00 q1+2 on the plane
pls tell me how bad I messed up
I finally contributed gas

### CODE REVIEW

**Comment by RedMincrafter** [comment]
how do I add tags and stuff
- ooh markdown
- help
# I need it

**Comment by AG-88301** [comment]
pls dont do multiple qs in one pull req

**Review by AG-88301 — CHANGES_REQUESTED** [review]
Add Python Solution to Q1 (gpt will do)

File: solutions/bio1/Ants/index.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+      <SSubtitle>Initialising the board</SSubtitle>
+      <SText>
+        Initalising the board is simple: an $11$x$11$ white grid. (I used a 2D Boolean List.)
+        Initalising the ants is also simple, input their positions and directions, and then parse them into usable data. (I subtract one for zero indexing, and change the direction into a number, which i use as an index to a list of directions.
+      </SText>
+
+      <SSubtitle>Mainloop</SSubtitle>
+      <SText>
+        Before the mainloop, the program takes in an initial value for $n$, then uses it in a while loop, which has the condition ($n$!=-1).
```
Comment (AG-88301) [inline solutions/bio1/Ants/index.tsx:44]: $n \neq -1$
```
$n \neq -1$
```

**Review by AG-88301 — CHANGES_REQUESTED** [review]

File: solutions/bio1/Ants/index.tsx (hunk 1 above)
Comment (RedMincrafter) [reply solutions/bio1/Ants/index.tsx]: did this thanks

**Review by RedMincrafter — COMMENTED** [review]
all changed done
- fixed latex
- correctly spelld initialise
- added passwords/sol.py

**Review by macaquedev — APPROVED** [review]
nice

**Pull request merged**